INSTALL.md

```diff
@@ -57,15 +57,15 @@ The script will ask you questions about your set up. Below are some suggestions:
   processes like Hadoop, Zookeeper, and the Accumulo client code.  If Accumulo
   worker processes are swapped out and unresponsive, they may be killed.
 
-After the `create-config` command is run, the `conf/` directory will contain
-`accumulo-env.sh`, `accumulo-site.xml`, and few a additional files. These files require
+After the `accumulo-config` command is run, the `conf/` directory will contain
+`accumulo.conf`, `accumulo-site.xml`, and few a additional files. These files require
 a few edits before starting Accumulo.
 
 ### Secret
 
 Accumulo coordination and worker processes can only communicate with each other
 if they share the same secret key.  To change the secret key set
-`instance.secret` in `conf/accumulo-site.xml`.  Changing this secret key from
+`instance.secret` in `accumulo-site.xml`.  Changing this secret key from
```

Comment:
It's still located in conf for the tarball distribution but I changed the manual to not include directory/path of the file as accumulo-site.xml could be in other locations like `/etc/accumulo` in downstream packaging.